docs/main/03-research/01-vendors/04-trello/04-trello.md

```diff
@@ -0,0 +1,198 @@
+- Change visibility:
+  - Private &rarr; Only board members can see this board.
+  - Workspace &rarr; All workspace members can view it.
+  - Public &rarr; Anyone on the internet can view it.
+
+- Finally, you can click on **Create** or you can **Start with a template** (which we discussed previously).
+
+### Inside the Board
+
+Welcome to your board’s page! Your workspace panel is still visible on the left, but you can close it if you prefer a cleaner view.
+
+![](photos/07-inside-the-board.jpg)
+
+We’ll explore this page from top to bottom, as it’s the most important part of Trello.
+
+#### 1. Workspace panel
+
+Here, we have the workspace panel again, with a few additional options:
+
+![](photos/08-workspace-panel.jpg)
+
+Section | Description
+:- | :-
+*More from Atlassian |  The icon on the far left (a 3x3 grid), which is primarily for advertising other Atlassian products.
+Search bar | Use this to find boards within your workspace.
+Notifications | See reminders, alerts, and other updates here.
+Information | This option leads to a Trello tour, which can be useful if you're new to Trello.
+Account | Access and modify your account settings here.
+
+#### 2. Board panel
+
+This section includes options specific to your board:
+
+![](photos/09-board-panel.jpg)
+
+Section | Description
+:- | :-
+Name | Double-click the board name to rename it.
+Star | Star your board to have it appear at the top of your boards list.
+Change visibility | Has already told.
+Customize views | The default view is the Kanban board, but Premium users can access additional views like Table, Calendar, Timeline, Dashboard, and Map. Only having the Kanban view can be a disadvantage if you’re looking for more layout options, which might make other tools appealing in comparison.
+Power-Ups | Add features and integrations like Google Drive or Slack to your board. Power-Ups work like extensions. E.g., using GitHub power-up, you can attach branches, commits, etc. directly to your cards.
+Automation | Will be explained later
+Filters | Filter cards using options like Members, Due date, Labels, Activity.
+Share | Share your board with others.
+Menu | A menu with options: Description for the board, See the Activity, Comments, Archived Items, Change Workspace, Set Permissions to the Members, Change Background, Automations, Power-Ups, Labels, Stickers, Watch, Email-to-board (add cards via email), Copy board, Print, export, share, Close the board.
+
+#### 3. View
+
+The default view here is a Kanban board, a visual tool for managing workflows and tracking tasks through stages like "To Do," "In Progress," and "Done." It helps teams quickly see the status of tasks and improve task flow.
+
+![](photos/10-view.jpg)
+
+The board consists of **Lists**. Each list represents a stage in the workflow. Typically, there are three lists: To Do, In Progress, and Done. However, you can rename, reorder, or add new lists by double-clicking on their names or dragging them.
+
```

Comment:
heading 4 of "Lists"

Reply:
The heading hierarchy has became kind of messy here! For example, here the "Lists" has been assigend a h6! I recommend you to look them up @atareversei